aggregator/README.md

```diff
@@ -131,7 +131,7 @@ This is done via comparing the `data_rlc` of `chunk_{i-1}` and ` chunk_{i}`.
 ![Dynamic_inputs](./figures/hash_table.jpg)
 
 
-Our keccak table uses `2^19` rows. Each keccak round takes `300` rows. When the number of round is is less than $2^19/300$, the cell manager will fill in the rest of the rows with dummy hashes. 
+Our keccak table uses `2^19` rows. Each keccak round takes `300` rows. When the number of round is is less than `2^{19}/300`, the cell manager will fill in the rest of the rows with dummy hashes. 
```

Comment:
do you want to use latex or code block? if latex, you need to use `$2^{19}/300$`. for code block, you don't need to add `{}`.